src/main/resources/messages.properties

```diff
@@ -578,15 +578,28 @@ screen.tosconsent.link.cancel=Cancel and go back to OSF
 # Institution login page
 #
 screen.institutionlogin.title=Institution SSO
-screen.institutionlogin.tips=Sign in through your institution
+screen.institutionlogin.heading=Sign in through your institution
 screen.institutionlogin.message.select=If your institution has partnered with OSF, please select its name below and sign in with your institutional credentials. If you do not currently have an OSF account, this will create one for you.
 screen.institutionlogin.message.auto=Your institution has partnered with OSF. Please continue to sign in with your institutional credentials. If you do not currently have an OSF account, this will create one for you.
 screen.institutionlogin.heading.select=Select your institution
 screen.institutionlogin.heading.auto=Your institution
 screen.institutionlogin.link.select=Not your institution?
+screen.institutionlogin.link.unsupported=I can't find my institution
 screen.institutionlogin.button.submit=Sign in
 screen.institutionlogin.osf=Sign in with your OSF account
 #
+# Unsupported Institution
+#
+screen.instnloginna.heading=I can't find my institution
+screen.instnloginna.message=Signing into the OSF with institutional credentials is enabled for <a href="{0}">OSF Institutions members</a>. If your institution is not yet an OSFI member, choose one of the following sign-in methods.
+screen.instnloginna.existing.heading=I already have an OSF account
+screen.instnloginna.existing.message=If you have already have an OSF password, you can sign in to the OSF
+screen.instnloginna.existing.button=Sign in with OSF
+screen.instnloginna.existing.setpw.message=If you have not yet set an OSF password for your account, create a password
+screen.instnloginna.existing.setpw.button=Set a password
+screen.instnloginna.existing.setpw.label.email=<span class="accesskey">E</span>mail
+screen.instnloginna.existing.setpw.label.email.accesskey=e
```

Comment:
I didn't change the "language" but renamed the names of variables here so that they follow similar pattern and they are not too long when used in the templates.

Reply:
Can I propose `unsupp-instn` as a compromise key?  I like the idea of using a shorter key, but `instnloginna` rubs me the wrong way.  I don't parse `loginna` as two phrases ("login" & "na"), but as a single nonsense word "loginna".  `instnloggin` also stikes me as odd for reasons I can't quite verbalize.  Maybe it's b/c there are too many consecutive consonants?  I think `institutionlogin` works b/c it is two entire words.  Once we start abbreviating them, that clarity disappears.

This isn't a dealbreaker for me, just a gut feeling.

Reply:
Updated and everything works as expected :)